app/controllers/links_controller.rb

```diff
@@ -30,6 +30,10 @@ def external_url(type:, options: {})
       guide, chapter, flavor = params.permit(:section, :chapter, :flavor).values_at(:section, :chapter, :flavor)
       flavor ||= self.class.new_docs_flavor
       docs_url(guide: guide, chapter: chapter, flavor: flavor)
+    when 'upgrade'
+      docs_url(guide: 'Upgrading_Project', flavor: self.class.new_docs_flavor)
```

Comment:
A bit of a nitpicking, but can we extract this to a separate method too:
```suggestion
      upgrade_url(params[:section])
```

and then have 
``` ruby
def upgrade_url(section)
  docs_url(guide: 'Upgrading_Project', flavor: self.class.new_docs_flavor)
end
```

This will clean up the branding

Reply:
That's even better, thanks!